leakcanary-android-benchmark/src/androidTest/res/values/bools.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <bool name="leak_canary_allow_in_non_debuggable_build">true</bool>
```

Comment:
why is this a resource?

Reply:
Benchmark library requires marking application as non-debuggable to remove the penalty of debug tools; and our new check crashes the initializer because it detects that build is not in debug :)

Reply:
https://github.com/square/leakcanary/pull/1752/files#diff-08ae653022e2f96a373a6d00777064e3R21

Reply:
I meant: it doesn't have to be a resource, this isn't published we don't need anyone to override, rihgt?

Reply:
ah, I read this wrong. Obviously we need this duh.